modules/drivers/snowflake/src/metabase/driver/snowflake.clj

```diff
@@ -670,7 +672,10 @@
   (sql-jdbc.execute/do-with-connection-with-options
    driver database nil
    (fn [^java.sql.Connection conn]
-     (with-open [stmt (.prepareStatement conn "show parameters like 'TIMEZONE' in user;")
+     (with-open [stmt (.prepareStatement conn (str "show parameters like 'TIMEZONE' in user"
+                                                   (when-let [user (get-in database [:details :user])]
+                                                     (str " \"" (str/replace user #"\s" "") "\""))
```

Comment:
This reminds me of https://xkcd.com/327/

I don't know what we want to achieve here, but it does look a bit dangerous.

Reply:
Github should have a badge for receiving a review comment with link to that xkcd. Reading your comment I've realized that change is probably redundant. When I confirm that, I'll revert this and other related bit. Thanks.